src/System.Text.JsonLab/System/Text/Json/JsonWriterUtf8.cs

```diff
@@ -565,12 +565,9 @@ public void WriteValue(Guid value)
         /// </summary>
         public void WriteNull()
         {
-            //TODO: Optimize, just like WriteValue(long value)
-            WriteItemSeperatorUtf8();
-            _firstItem = false;
-            WriteSpacingUtf8();
-
-            WriteJsonValue(JsonConstants.NullValue);
+            ReadOnlySpan<byte> valueSpan = JsonConstants.NullValue;
+            int bytesNeeded = CalculateValueBytesNeeded(valueSpan, sizeof(byte), 2);
```

Comment:
Similarly, here, CalculateValueBytesNeeded is adding the bytes needed to add the quotes, which shouldn't be needed here.